src/internal/ppsutil/util.go

```diff
@@ -467,6 +467,7 @@ func FindPipelineSpecCommitInTransaction(txnCtx *txncontext.TransactionContext,
 	if err != nil {
 		return nil, errors.EnsureStack(err)
 	}
+	curr = commitInfo.Commit
```

Comment:
Is this a similar concern to this: https://github.com/pachyderm/pachyderm/pull/8306#discussion_r1070703558?

Reply:
yep same thing

Reply:
We should have a TODO for this as well.